src/presentation/ui/component/databases/installedDatabaseTypeSummaryTable.templ

```diff
@@ -0,0 +1,56 @@
+package componentDatabases
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	"strconv"
+)
+
+templ InstalledDatabaseTypeSummaryTable(
+	selectedDatabaseTypeDatabases []entity.Database,
+) {
+	<table class="w-full table-auto border-collapse rounded-md transition-all duration-300 ease-in-out">
+		<thead class="text-xs uppercase text-neutral-400">
+			<tr class="border-b border-neutral-500 border-opacity-90 text-center">
+				<th scope="col" class="px-5 py-3 w-1/7 text-left">Database Name</th>
```

Comment:
If you're not going to use Database prefix on Users, you may as well just use Name here I think.

Reply:
Done!